.github/workflows/tests.yaml

```diff
@@ -16,6 +16,8 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - name: conventional commits
+        uses: webiny/action-conventional-commits@v1.1.0
```

Comment:
Hmm actually, thinking more about this is this the place we want to enforce it? If we're squashing commits, the messages within a PR don't really matter — just the commit message that we end up using in the PR. 

Perhaps what we want is this? https://github.com/marketplace/actions/conventional-pr-title

Reply:
We probably also want that. To be fair, we also do _rebase and merge_, the _squash_ usually happens when the PR is not neat or functional enough

Reply:
I was looking into adding that, but not comfortable with the write permission it sets up, FWIW this one has more stars https://github.com/marketplace/actions/semantic-pull-request

In any case, at time of _merge_ or _squash merge_ the commit can still be edited, so whoever does this needs to be on their best behavior :see_no_evil: